<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1659603981290' stop='1659604005356'>
    <name>e-shop login, test case: 2</name>
    <title>e-shop login, test case: 2</title>
    <test-cases>
        <test-case start='1659603981290' status='passed' stop='1659603985042'>
            <name>should open main page</name>
            <title>should open main page</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
        </test-case>
        <test-case start='1659603985070' status='passed' stop='1659603993696'>
            <name>should open login page</name>
            <title>should open login page</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
        </test-case>
        <test-case start='1659603993713' status='passed' stop='1659604001127'>
            <name>should input login data and sign in</name>
            <title>should input login data and sign in</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
        </test-case>
        <test-case start='1659604001142' status='failed' stop='1659604005299'>
            <name>should check if logged correctly</name>
            <title>should check if logged correctly</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
            <failure>
                <message>Timed out retrying after 4000ms: Expected to find content: 'My account' within the element: &lt;h1.page-heading> but never did.</message>
                <stack-trace>AssertionError: Timed out retrying after 4000ms: Expected to find content: 'My account' within the element: &lt;h1.page-heading> but never did.
                    at Function.checkIfLogged (http://automationpractice.com/__cypress/tests?p=cypress\e2e\automation_practice\task-1.cy.js:154:29)
                    at Context.eval (http://automationpractice.com/__cypress/tests?p=cypress\e2e\automation_practice\task-1.cy.js:122:28)</stack-trace>
            </failure>
        </test-case>
    </test-cases>
</ns2:test-suite>